packages/cluster/src/common.js

```diff
@@ -32,8 +41,29 @@ module.exports.initDockerComposeCache = async ({
     return null
 }
 
-module.exports.makeComposeProject = version => `d2-cluster-${version}`
-module.exports.makeDockerImage = version => `${version}-alpine`
+module.exports.writeCache = (cache, loc) => {
+    try {
+        const target = path.dirname(loc)
+        fs.writeFileSync(
+            path.join(target, 'clusterCache.json'),
```

Comment:
Probably good to factor the filename out as a const

Reply:
Will do. ⚡️

Reply:
b727e65